docker/README.md

```diff
@@ -29,6 +32,8 @@ docker run --rm -it -v /path/to/synapse\:/src:ro -v /path/to/where/you/want/logs
 
 The following environment variables can be set with `-e` to control the test run:
 
+ * `SYTEST_BRANCH`: controls which branch of synapse or dendrite we fetch when  `/src`
+   is missing. If omitted or if that branch doesn't exist, we'll fetch the  appropriate development branch.
```

Comment:
this isn't quite right. As above: if `/src` is missing, I don't think it works at all...

Reply:
Argh! Thanks. I think I keep confusing `/src` and `/sytest`. It's the CI job that has to fetch the source

Reply:
No worries! It is pretty confusing.